botocore/parsers.py

```diff
@@ -507,18 +507,18 @@ def _handle_timestamp(self, shape, value):
         return self._timestamp_parser(value)
 
     def _do_error_parse(self, response, shape):
-        body = json.loads(response['body'].decode(self.DEFAULT_ENCODING))
-        error = {"Error": {}, "ResponseMetadata": {}}
+        body = self._parse_body_as_json(response['body'])
+        error = {"Error": {"Message": '', "Code": ''}, "ResponseMetadata": {}}
```

Comment:
I feel like you have pushed back on this before where if the Message or Code cannot be found we set a default empty value because it hides up a possible problem in our JSON error parsing. Am I remembering correctly or not?

I am personally fine with having the empty default value because having a runtime AttributeNotFound/KeyNotFound error is a pretty bad experience. But just checking that you are fine with it.